src/main/java/frc/robot/RobotContainer.java

```diff
@@ -178,44 +181,60 @@ private void initCommands() {
+      // Intake
+      if (intakeForwardCommand != null && intakeReverseCommand != null) {
+        buttons[Constants.RBumper].whileHeld(intakeForwardCommand);
+        buttons[Constants.RTriggerButton].whileHeld(intakeReverseCommand);
+      }
 
-    // Shooter
-    if (shooterSubsystem != null && shooterHeld != null) {
-      buttons[Constants.backButton].whenPressed(shooterHeld);
-      buttons[Constants.LJoystickButton].whenPressed(
-          new InstantCommand(shooterSubsystem::cycleAimModeNext, shooterSubsystem));
-      buttons[Constants.RJoystickButton].whenPressed(
-          new InstantCommand(shooterSubsystem::cycleAimModePrevious, shooterSubsystem));
-    }
+      // Shooter
+      if (shooterSubsystem != null && shooterHeld != null) {
+        buttons[Constants.backButton].whenPressed(shooterHeld);
+        buttons[Constants.LJoystickButton].whenPressed(
+            new InstantCommand(shooterSubsystem::cycleAimModeNext, shooterSubsystem));
+        buttons[Constants.RJoystickButton].whenPressed(
+            new InstantCommand(shooterSubsystem::cycleAimModePrevious, shooterSubsystem));
+      }
 
-    //CDS
-    if (CDSSubsystem != null) {
-      CDSForwardCommand = new CDSForwardCommand(CDSSubsystem);
-      CDSSubsystem.setDefaultCommand(new CDSAutoAdvanceCommand(CDSSubsystem));
-      //CDSReverseCommand = new CDSReverseCommand(CDSSubsystem, shooterSubsystem);
-      //CDSSubsystem.senseColor();
-    }
-    
-    if (CDSForwardCommand != null && outtakeCommand != null) {
-      buttons[Constants.LTriggerButton].whileHeld(CDSForwardCommand);
-      //buttons[Constants.RTriggerButton].whileHeld(CDSReverseCommand);
-      buttons[Constants.RTriggerButton].whileHeld(outtakeCommand);
-    }
+      // CDS
+      if (CDSSubsystem != null) {
+        CDSForwardCommand = new CDSForwardCommand(CDSSubsystem);
+        CDSSubsystem.setDefaultCommand(new CDSAutoAdvanceCommand(CDSSubsystem));
+      }
 
-    // Limelight
-    if (limelightAlign != null) {
-      buttons[Constants.startButton].whenPressed(limelightAlign);
-    }
+      if (CDSForwardCommand != null && outtakeCommand != null) {
+        buttons[Constants.LTriggerButton].whileHeld(CDSForwardCommand);
+        buttons[Constants.RTriggerButton].whileHeld(outtakeCommand);
+      }
 
-    if (climbSubsystem != null) {
-      if (!Constants.oneController) {
-        buttons2[Constants.startButton].whenPressed(
-            new InstantCommand(climbSubsystem::toggleClimbEnable, climbSubsystem));
+      // Limelight
+      if (limelightAlign != null) {
+        buttons[Constants.startButton].whenPressed(limelightAlign);
+      }
+
+      if (climbSubsystem != null) {}
+    } else {
+
+      if (shooterSubsystem != null && shooterHeld != null) {
+        buttons[Constants.LBumper].whileHeld(shooterHeld);
+        buttons[Constants.LTriggerButton].whileHeld(
+            shooterHeld.beforeStarting(
+                () -> shooterSubsystem.setAimMode(Constants.AimModes.LOW.ordinal())));
```

Comment:
Lets remove the beforeStarting stuff. Its not the best way to do this. A parameter should get added to shooterHeld for the aimMode @Majorferguson21